roles/provision-asb-installer/defaults/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+namespace: "ansible-service-broker"
+service_account_name: "asb"
+client_service_account_name: "asb-client"
+auth_cluster_role_name: "asb-auth"
+access_cluster_role_name: "asb-access"
+deploy_broker_template_url: https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml
```

Comment:
I would like for us to, down the road, move away from the template but I think this is fine for now.

Reply:
I agree and would like to use the new Ansible module when the PR gets merged. https://github.com/ansible/ansible/pull/33517